src/packages/core/property-editor/uis/tree-picker/property-editor-ui-tree-picker.element.ts

```diff
@@ -60,7 +101,7 @@ export class UmbPropertyEditorUITreePickerElement extends UmbLitElement implemen
 	render() {
 		return html`<umb-input-tree
 			.value=${this.value}
-			.type=${this.type}
+			.type=${this.type ?? 'content'}
```

Comment:
Looks like the tree picker already defaults to 'content'. Consider if you really need to specify it here as well.